code/__DEFINES/dcs/signals.dm

```diff
@@ -264,6 +264,12 @@
 #define COMSIG_MOVABLE_BUCKLE "buckle"
 ///from base of atom/movable/unbuckle_mob(): (mob, force)
 #define COMSIG_MOVABLE_UNBUCKLE "unbuckle"
+///from /atom/movable/proc/buckle_mob, used by the ridable element to see if someone can successfully ride the target
```

Comment:
```suggestion
///from base of atom/movable/proc/buckle_mob(): (mob/living/M, force, check_loc, addl_ride_flags)
```
Signals are generic events, describe them generically whenever possible.
The current application is not important, that will change with time and this won't likely be updated.